api/MediaStreamTrack.json

```diff
@@ -849,10 +849,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
Ghost feature?

Reply:
There does seem to be a lot of these ghost features that we're finding...  I know that @saschanaz has been doing a little project to narrow down and remove these features without support!

Reply:
This one's in https://w3c.github.io/webrtc-identity/identity.html#isolated-track, but I guess not implemented.